README.md

```diff
@@ -3,25 +3,49 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
 
 More info on the Mapping Viz Tools [wiki](https://github.com/zooniverse/mapping-viz-tools/wiki/What-are-Azure-Functions%3F)
 
+## Getting started
 ### Dependencies
 In order to run this code, you'll need to install the Azure Functions [core tools.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash#v2)
 
 Although not required, you may also want to install the Azure VS Code [extension.](https://code.visualstudio.com/docs/azure/extensions)
 
-### yarn install
+### Local database setup
+1. Install PostgreSQL 9.5 and run the PostgreSQL service locally 
+2. Create a database called `mapping_viz_local`. You can do so with the following command on the terminal:  
```

Comment:
if you move to a docker solution for running postgres server this will have to be a docker-compose run cmd (run the cmd in the container).
```
docker-compose run --rm postgres createdb mapping_viz_local
```